book/doctrine.rst

```diff
@@ -641,6 +641,11 @@ to easily fetch objects based on multiple conditions::
 
     If you click the icon, the profiler will open, showing you the exact
     queries that were made.
+    
+    If you make more than 50 queries on a page, the icon will turn yellow to
+    let you know that you are making a lot of requests to the database. This
+    is only a warning and does not have any effect on the execution of further
+    queries.
```

Comment:
Sure, sounds great to me. I was mainly concerned with getting the information about the yellow icon in there.